app/controllers/api/v1/admin/tags_controller.rb

```diff
@@ -0,0 +1,77 @@
+# frozen_string_literal: true
+
+class Api::V1::Admin::TagsController < Api::BaseController
+  include Authorization
+  before_action -> { authorize_if_got_token! :'admin:read' }, only: [:index, :show]
+  before_action -> { authorize_if_got_token! :'admin:write' }, only: :update
```

Comment:
It probably would make sense to have finer permissions, e.g. `admin:read:tags` and `admin:write:tags`, but that can be changed later.